README.md

```diff
@@ -14,9 +14,9 @@
 
 # Build time aware visualizations of multimodal data
 
-Use the Rerun SDK (available for C++, Python and Rust) to log data like images, tensors, point clouds, and text. Logs are streamed to the Rerun Viewer for live visualization or to file for later use.
+Use the Rerun SDK (available for C++, Python and Rust) to log data like images, tensors, point clouds, and text. Logs are streamed to the Rerun viewer for live visualization or to file for later use.
 
-* [Run the Rerun Viewer in your browser](https://www.rerun.io/viewer)
+* [Run the Rerun viewer in your browser](https://www.rerun.io/viewer)
```

Comment:
We capitalize product names, and "Rerun Viewer" counts as a product (currently). See `DESIGN.md`

Reply:
…though I see we do so very inconsistently and don't enforce it in lint.py

Reply:
handled in https://github.com/rerun-io/rerun/pull/6317